packages/@aws-cdk/aws-rds/README.md

```diff
@@ -82,9 +73,9 @@ your instances will be launched privately or publicly:
 
 ```ts
 const instance = new rds.DatabaseInstance(this, 'Instance', {
-  engine: rds.DatabaseInstanceEngine.ORACLE_SE1,
+  engine: rds.DatabaseInstanceEngine.postgres({ version: rds.PostgresEngineVersion.VER_12_3 }),
```

Comment:
Let's maybe use a newer Oracle here...?

Reply:
Sure, can do. Kind of curious why, but happy to swap two examples back.